lib/auth/webauthnwin/api_test.go

```diff
@@ -27,6 +28,11 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+func init() {
+	// Make tests silent.
```

Comment:
I think we normally allow tests to be loud, the testing frameworks should deal with that.

Reply:
I have follow the same pattern that we have for touch_id https://github.com/gravitational/teleport/blob/57508a9e58613201ee970be762f1c41dc38bed61/lib/auth/touchid/api_test.go#L41-L44

Reply:
Indeed, but it is an outlier itself :-) but this is more of a "nit", i.e. a small thing, nothing to truly worry about.

Reply:
I like silent tests, but then again I'm to blame for the original snippet... :)